src/equations/compressible_euler_2d.jl

```diff
@@ -884,6 +884,85 @@ function flux_hllc(u_ll, u_rr, orientation::Integer, equations::CompressibleEule
 end
 
 
+"""
+    flux_hlle(u_ll, u_rr, orientation, equations::CompressibleEulerEquations2D)
+
+Computes the HLLE (Harten-Lax-van Leer-Einfeldt) flux for the compressible Euler equations.
+Special estimates of the signal velocites and linearization of the Riemann problem developed
+by Einfeldt to ensure that the internal energy and density remain positive during the computation
+of the numerical flux.
+
+- Bernd Einfeldt (1988)
+  On Godunov-type methods for gas dynamics.
+  [DOI: 10.1137/0725021](https://doi.org/10.1137/0725021)
+- Bernd Einfeldt, Claus-Dieter Munz, Philip L. Roe and Björn Sjögreen (1991)
+  On Godunov-type methods near low densities.
+  [DOI: 10.1016/0021-9991(91)90211-3](https://doi.org/10.1016/0021-9991(91)90211-3)
+"""
+function flux_hlle(u_ll, u_rr, orientation::Integer, equations::CompressibleEulerEquations2D)
```

Comment:
Is this covered by coverage? If the elixir is not, do we need a unit test/smoke test for this?

Reply:
I was thinking about this, too. It's at least not tracked in coverage since it's only used in the elixir that just runs without coverage. We could add some simple unit tests of the form `@test numerical_flux(u, u, orientation, equations) ≈ flux(u, orientation, equations)`, e.g., somewhere near 
https://github.com/trixi-framework/Trixi.jl/blob/8c8b43ce7f0e4f2f0fa7cb50d2fe6d4c6aa8c575/test/test_unit.jl#L553

Reply:
I took a swing at adding such a small unit test that checks for flux consistency